packages/providence/src/cli/generate-extend-docs-data.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const fs = require('fs');
+const pathLib = require('path');
+const { performance } = require('perf_hooks');
+
+const { providence } = require('../program/providence.js');
+const { QueryService } = require('../program/services/QueryService.js');
+const { LogService } = require('../program/services/LogService.js');
+
+async function launchProvidenceWithExtendDocs(referencePaths, prefixObj, outputFolder) {
+  const t0 = performance.now();
+
+  const results = await providence(
+    QueryService.getQueryConfigFromAnalyzer('match-paths', { prefix: prefixObj }),
+    {
+      gatherFilesConfig: {
+        extensions: ['.js', '.html'],
+        excludeFolders: ['coverage', 'test'],
+      },
+      queryMethod: 'ast',
+      report: false,
+      targetProjectPaths: [pathLib.resolve(process.cwd())],
+      referenceProjectPaths: referencePaths,
+    },
+  );
+
+  const outputFilePath = pathLib.join(outputFolder, 'providence-extend-docs-data.json');
```

Comment:
Currently this file already exists with an empty array in it, is that intended?

Reply:
Nope :) Will remove it